### ISSUE TITLE
Design --> Script

### ISSUE THREAD

**Issue by kylekesses** [mecs13/website_text#129]
I understand this to be close to what we printed and mailed to Dwellworks @mecs13

**Comment by mecs13** [comment]
yeah we should calendar for a an actual script sample and graphic to be put here
